migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/ValueTypeLookup.java

```diff
@@ -751,6 +781,10 @@ public void resolveAll(Library library) throws IOException {
         if (ld == null) throw new IOException("LibraryDesign not found");
         lai.setLibraryDesign(ld);
       }
+
+      LibraryDesignCode ldc = resolve(lai.getLibraryDesignCode());
+      if (ldc == null) throw new IOException("LibraryDesignCode not found");
```

Comment:
Oh, I misread this. Thought it was a case of `lai.getLibraryDesignCode()` being null. Disregard my comment